backend/pkg/api/data_access/vdb_rewards.go

```diff
@@ -678,7 +677,7 @@ func (d *DataAccessService) GetValidatorDashboardGroupRewards(ctx context.Contex
 		ret.ProposalClSlashingIncReward = ret.ProposalClSlashingIncReward.Add(entry.SlasherRewards.Mul(gWei))
 	}
 
-	ret.Proposal.Income = ret.Proposal.Income.Add(elRewards)
+	ret.Proposal.Income = ret.Proposal.Income.Add(d.convertElToMain(elRewards).Mul(decimal.NewFromInt(d.config.Frontend.ElCurrencyDivisor)))
```

Comment:
Issue: You're converting the EL currency to the main currency and adding it to a field that contains values in the CL currency. On Gnosis, this would mean that you've converted xDai to GNO and then added it to a mGNO value. This field should be a `ClElValue` struct.